addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -51,8 +51,8 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
     def _cart_update(self, product_id=None, line_id=None, add_qty=0, set_qty=0, **kwargs):
         values = {}
         product = self.env['product.product'].browse(product_id)
-        if product.type == 'event_booth' and set_qty > 1:
+        if product.detailed_type == 'event_booth' and set_qty > 1:
```

Comment:
Would make a specific commit linking the commit from Wan that broke the type / detailed_type (https://github.com/odoo/odoo/commit/3d7b9bbf27298dc7f2f7e61a13120988f9bd422a ) as this is a bugfix (we will backport it quickly).